config/jobs/kubernetes/test-infra/test-infra-trusted.yaml

```diff
@@ -221,6 +221,39 @@ postsubmits:
     rerun_auth_config:
       github_users:
       - alvaroaleman
+  - name: post-test-infra-push-prow-staging
+    cluster: test-infra-trusted
+    # Runs on more than just the Prow dir to include some additional images that we publish to gcr.io/k8s-prow.
+    run_if_changed: '^(WORKSPACE|load.bzl|repos.bzl|containers.bzl|prow|ghproxy|label_sync/.+\.go|robots/commenter|robots/pr-creator|robots/issue-creator|testgrid/cmd|gcsweb)'
```

Comment:
This is no longer accurate. What is the plan for all these images?

Reply:
updated to be more accurate